Library/Homebrew/cmd/audit.rb

```diff
@@ -8,6 +8,13 @@ def audit
     formula_count = 0
     problem_count = 0
 
+    strict = ARGV.delete "--strict"
+    if strict && ARGV.any?
```

Comment:
Can this be done by checking `ARGV.formulae.any?` instead of deleting from ARGV?